crates/holochain_conductor_api/src/config/conductor/keystore_config.rs

```diff
@@ -30,7 +32,7 @@ pub enum KeystoreConfig {
         /// "lair-keystore-config.yaml" file.
         /// If not specified, will default to the ConductorConfig
         /// `[environment_path]/ks`.
-        #[serde(default, skip_serializing_if = "Option::is_none")]
+        #[schemars(default, skip_serializing_if = "Option::is_none")]
```

Comment:
Why does this need to change? Won't that now mean that Serde won't skip serializing for `None`s?